src/main/java/com/raeic/embarker/land/schedulers/LandScheduler.java

```diff
@@ -0,0 +1,144 @@
+                    Class<?> EnumWorldBorderAction = PacketPlayOutWorldBorder.getDeclaredClasses()[0];
+                    Constructor<?> PacketPlayOutWorldBorderConstructor = PacketPlayOutWorldBorder.getConstructor(
+                            WorldBorder,
+                            EnumWorldBorderAction
+                    );
+
+                    Class<?> EntityPlayer = Globals.reflectionUtil.getNMSClass("EntityPlayer");
+                    Field EntityPlayer_playerConnection = EntityPlayer.getDeclaredField("playerConnection");
+
+                    Class<?> PlayerConnection = Globals.reflectionUtil.getNMSClass("PlayerConnection");
+                    Class<?> Packet = Globals.reflectionUtil.getNMSClass("Packet");
+                    Method PlayerConnection_sendPacket = PlayerConnection.getMethod("sendPacket", Packet);
+
+                    // Begin building a world border
+                    Object craftPlayer = CraftPlayer.cast(p);
+                    Object worldBorder = WorldBorderConstructor.newInstance();
+
+                    Object worldServer = CraftWorld_getHandle.invoke(CraftWorld.cast(p.getWorld()));
+                    WorldBorder_world.set(worldBorder, WorldServer.cast(worldServer));
+
+                    boolean playerIsInChunk = chunk.getX() == metadata.getChunk().getCoordX() &&
+                                              chunk.getZ() == metadata.getChunk().getCoordZ();
+
+                    if (metadata.getDuration() > 0 && playerIsInChunk) {
+                        WorldBorder_setSize.invoke(worldBorder, 17);
+                        WorldBorder_setCenter.invoke(worldBorder, (chunk.getX() * 16) + 8, (chunk.getZ() * 16) + 8);
+
+                        metadata.setDuration(metadata.getDuration() - PLAYER_VIEWING_BORDER_SCHEDULER_TICKS);
+                    } else {
+                        // Set border to 599999968 to disable it
+                        // https://minecraft.gamepedia.com/World_border#Commands
+                        WorldBorder_setSize.invoke(worldBorder, 59999968);
+                        WorldBorder_setCenter.invoke(worldBorder, 0, 0);
+
+                        playersToRemove.add(p);
+                    }
+
+                    Object packet = PacketPlayOutWorldBorderConstructor.newInstance(
+                            worldBorder,
+                            EnumWorldBorderAction.getField("INITIALIZE").get(null)
+                    );
+
+                    Object entityPlayer = EntityPlayer.cast(CraftPlayer_getHandle.invoke(craftPlayer));
+                    Object playerConnection = PlayerConnection.cast(EntityPlayer_playerConnection.get(entityPlayer));
+                    PlayerConnection_sendPacket.invoke(playerConnection, packet);
+
+                } catch (NoSuchMethodException | NoSuchFieldException | IllegalAccessException | InstantiationException | InvocationTargetException e) {
+                    e.printStackTrace();
```

Comment:
I would prefer to catch these during plugin initialization and throw as an error, rather than a warning